CodeEdit/Features/Acknowledgements/Views/AcknowledgementsView.swift

```diff
@@ -64,6 +65,12 @@ struct AcknowledgementRow: View {
     }
 }
 
+struct Ackn_Previews: PreviewProvider {
```

Comment:
Any reason for the shortened name of the struct?

Reply:
This was just for debugging and it has been removed in [d21f589](https://github.com/CodeEditApp/CodeEdit/pull/963/commits/d21f589d91bc63e96a04d519d054a46ff23f1716)